src/main/scala/org/camunda/feel/syntaxtree/Val.scala

```diff
@@ -186,6 +179,34 @@ case class ValYearMonthDuration(value: YearMonthDuration) extends Val {
 }
 
 case class ValDayTimeDuration(value: DayTimeDuration) extends Val {
+  override def toString: String = {
+    val millis = value.toMillis
+    val positive = millis > 0
+    val timeDuration = if (! positive) {
+      java.time.Duration.ofMillis(millis).negated()
+    } else {
+      java.time.Duration.ofMillis(millis)
+    }
+    val day = if(timeDuration.toDays != 0){
+      BigInteger.valueOf(timeDuration.toDays)
+    } else { null }
+    val hour = if(timeDuration.toHours % 24 != 0){
+      BigInteger.valueOf(timeDuration.toHours % 24)
+    } else { null }
+    val minute = if(timeDuration.toMinutes % 60 != 0){
+      BigInteger.valueOf(timeDuration.toMinutes % 60)
+    } else { null }
+    val seconds = if(timeDuration.getSeconds % 60 != 0){
+      BigInteger.valueOf(timeDuration.getSeconds % 60)
+    } else { null }
```

Comment:
I did this here since it is available in java 9, and not java 8, which is what I use when developing scala. I will of cause update if you are using java 11?

Reply:
@kaaquist good point. :+1: We need to support also Java 8. :sweat_smile: